p2panda-core/src/extensions.rs

```diff
@@ -1,9 +1,15 @@
 // SPDX-License-Identifier: AGPL-3.0-or-later
 
+use std::fmt::Debug;
+
 use serde::{Deserialize, Serialize};
 
 use crate::Header;
 
+pub trait Extensions: Clone + Debug + Default + Send + Sync {}
```

Comment:
I'd welcome any ideas / input regarding a doc comment for `Extensions`.

Reply:
Something like: "Extensions can be used to define custom fields in the operation header"?